src/js/Constants.js

```diff
@@ -38,11 +38,11 @@ export const MONTH_NAMES = [
 ]
 
 export const WEEKDAY_NAMES = [
-  'Sunday',
-  'Monday',
-  'Tuesday',
-  'Wednesday',
-  'Thursday',
-  'Friday',
   'Saturday',
+  'Friday',
+  'Thursday',
+  'Wednesday',
+  'Tuesday',
+  'Monday',
+  'Sunday',
 ]
```

Comment:
This is interesting I never thought to just reverse the order in here. Doesn't the table show up in the wrong order in this case?

Reply:
Monday on the top.
I think it show up right